src/github/handlers/issue-comment-created.ts

```diff
@@ -0,0 +1,36 @@
+import { GitHubContext } from "../github-context";
+import { getConfig } from "../utils/config";
+
+export default async function issueCommentCreated(context: GitHubContext<"issue_comment.created">) {
+  const body = context.payload.comment.body.trim();
+  if (/^\/help$/.test(body)) {
+    const comments = [
+      "### Available Commands\n\n",
+      "| Command | Description | Example |",
+      "|---|---|---|",
+      "| `/help` | List all available commands. | `/help` |",
+    ];
+    const configuration = await getConfig(context);
+    for (const pluginArray of Object.values(configuration.plugins)) {
+      for (const plugin of pluginArray) {
+        // Only show plugins that have commands available for the user
+        if (plugin.command) {
+          comments.push(`| \`${getContent(plugin.command)}\` | ${getContent(plugin.description)} | \`${getContent(plugin.example)}\` |`);
+        }
+      }
+    }
+    await context.octokit.issues.createComment({
+      body: comments.join("\n"),
+      issue_number: context.payload.issue.number,
```

Comment:
@gentlementlegen I can't understand how it works both in github issue comments and PR comments. So if the `/help` command is called from github issue comments then `context.payload.issue.number` contains issue number. If the `/help` command is called from a PR then `context.payload.issue.number` contains a PR number. Correct?

Reply:
Good remark. This is because for GitHub API, `issues` and `pull-request` are considered the same when filling the payload. Here is a QA test for `pull-request` help command: https://github.com/Meniole/ubiquibot-kernel/pull/2#issuecomment-2176318035